app/ChordNode.js

```diff
@@ -1025,7 +1034,7 @@ class ChordNode {
   /**
    * Placeholder for data migration within the joinCluster() call.
    */
-  async migrateKeys() {}
+  async migrateKeysAfterJoin() {}
```

Comment:
Since we're treating this as an abstract method, I did a bit of googling, and this seems like a good approach. Just immediately throw an error saying that this needs to be overridden in the subclass: https://medium.com/@yuribett/javascript-abstract-method-with-es6-5dbea4b00027

Reply:
I included the not implemented error, I like the approach.